src/commands/rover/RestartCommand.js

```diff
@@ -1,5 +1,6 @@
 const Accolades = require('../../Accolades.json')
 const Command = require('../Command')
+const Discord = require('discord.js')
```

Comment:
```suggestion
const { ShardClientUtil } = require('discord.js')
```

I would prefer object destructuring over importing everything here.